.github/workflows/tests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         env:
           AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Install Tox and any other packages
-        run: pip install tox wheel setuptools pipenv
+        run: pip install tox==3.15.0
```

Comment:
Shouldn't this be in `semgrep/Pipfile`?

Reply:
it is in the Pipfile. But tox runs pipenv to install packages into the environment, so on CI you get into a bit of a chicken and egg situation.